app/main.py

```diff
@@ -1 +1,54 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    FrontendDeveloper_skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
The `FrontendDeveloper_skills` should be a class attribute, but it's not being used correctly in the `__init__` method. Consider using `self.__class__.FrontendDeveloper_skills` to ensure it's accessed correctly as a class attribute.